proto/BUILD

```diff
@@ -2,9 +2,9 @@ sh_cmd(
     name = "generate",
     cmd = "cp $(out_location //proto/lucidity:_lucidity#protoc_go) proto/lucidity && cp $(out_location //proto/purity:_purity#protoc_go) proto/purity && cp $(out_location //proto/mettle:_mettle#protoc_go) proto/mettle",
```

Comment:
This and the .pb.gp files do not need review. I regenerated them to make the linter happy